cmd/proxy/.env

```diff
@@ -1,4 +1,2 @@
 ATHENS_MONGO_CONNECTION_STRING=mongodb://127.0.0.1:27017
 ATHENS_STORAGE_TYPE=mongo
-GO_ENV=test_postgres
```

Comment:
keep the test, this is passed to buffalo and we use ENV to decide whether or not it is development so we can spin up buffalos ParameterLogger. and if it's not set it will be considered development even for tests.